exporters/zipkin/src/main/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporter.java

```diff
@@ -76,25 +75,15 @@ public CompletableResultCode export(Collection<SpanData> spanDataList) {
       encodedSpans.add(encoder.encode(zipkinSpan));
     }
 
-    CompletableResultCode result = new CompletableResultCode();
-    sender
-        .sendSpans(encodedSpans)
-        .enqueue(
-            new Callback<Void>() {
-              @Override
-              public void onSuccess(Void value) {
-                exporterMetrics.addSuccess(numItems);
-                result.succeed();
-              }
-
-              @Override
-              public void onError(Throwable t) {
-                exporterMetrics.addFailed(numItems);
-                logger.log(Level.WARNING, "Failed to export spans", t);
-                result.fail();
-              }
-            });
-    return result;
+    try {
+      sender.send(encodedSpans);
```

Comment:
I believe this changes the implementation from being non-blocking to blocking. In practice, this is fine because BatchSpanProcess / SImpleSpanProcessor never calls `SpanExporter#export` concurrently.

Reply:
yep it does make it explicitly blocking (depending on sender some would have sneakily blocked!) Thanks for the notes on impact